.github/workflows/preview-comment.yml

```diff
@@ -0,0 +1,31 @@
+name: Comment on PR
+
+on:
+  pull_request:
+    branches:
+      - trunk
```

Comment:
This would cause the comment to be posted multiple times, whenever a PR is updated.

I'd change it something like this:

```
on:
  pull_request_target:
    types: [ opened ]
    branches:
      - trunk
```